docs/painless/painless-contexts/painless-context-examples.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[painless-context-examples]]
+=== Context examples
+
+Use the commands below to setup an Elasticsearch cluster to execute the example
+scripts for each context where an example is provided. Each example must be
+executed in the order of the table in the
+<<painless-contexts, previous section>> to work correctly.
+
+
+. Download the
+https://download.elastic.co/demos/painless/contexts/seats.json[seat data]
+for use in the examples. The data is a fictional set of plays and their
+locations and times where each individual document represents a single seat in
+a theater for a single play.
++
```

Comment:
In general, I try to avoid "meta data" like "for use in the examples" unless it's really needed for clarification. If you set the stage in the intro, "for use in the examples" is redundant. 

Since you've already presented this as example data, there's really no need to call it out as "fictional". I'd just say, "this data set contains booking information for a collection of plays. Each document represents a single seat for a play at a particular theater on a specific date and time."

Reply:
Fixed.